ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -746,7 +750,10 @@ DFSS2X:
   description: "Two discharge fan run command and status (start/stop). With dampers"
   is_abstract: true
   implements:
+  opt_uses:
```

Comment:
The alignment got messed up here - please fix.

Reply:
changed